app/router.ts

```diff
@@ -18,8 +19,13 @@ const Router = EmberRouter.extend({
         scheduleOnce('afterRender', this, () => {
             const page = this.get('url');
             const title = this.getWithDefault('currentRouteName', 'unknown');
+            const authenticated = this.get('session.isAuthenticated') ? 'Logged in' : 'Logged out';
+            const isPublic = 'true'; // This will have to be changed when we have private things
+            const metrics = this.get('metrics');
 
-            this.get('metrics').trackPage({ page, title });
+            metrics.trackPage({
+                page, title, dimension1: authenticated, dimension2: 'undefined', dimension3: isPublic,
```

Comment:
Since the dimensions are numbered when they're created, this can be error prone. The GA admin would have to create them in the exact order for each environment. This should defined in the ember config, so that it can be changed per environment, if needed.

In config/environment:
```js
ENV.metrics.dimensions = { // would probably put this somewhere in the ember-metrics GA config
    authenticated: 'dimension1',
    isPublic: 'dimension2',
}
```

in app/router:
```js
// Use object destructuring to grab the properties
const {
    authenticated,
    isPublic,
} = config.metrics.dimensions;

// Use computed keys for the dimension names
this.get('metrics').trackPage({
    page,
    title,
    [authenticated]: (!!this.get('session.isAuthenticated')).toString(), // 'true' or 'false'
    [isPublic]: 'true',
});
```